### DISCUSSION TITLE
Is there a starter template 'library'?

### DISCUSSION THREAD

**Discussion by njames** [tighten/jigsaw#638] [Q&A]
I see https://builtwithjigsaw.com/ but is there a list or [starter templates](https://jigsaw.tighten.com/docs/starter-templates/) somewhere?

I tried to look, is  my google fu that bad?

**Comment by damiani** [comment]
We have two starter templates to get you going, one for a blog and one for a docs site. The page you linked to includes instructions on how to use one of them as a starting point (`vendor/bin/jigsaw init blog`) and there are links on that page to view previews of each of the templates ([blog preview](https://jigsaw-blog-staging.tighten.co/), [docs preview](https://jigsaw-docs-staging.tighten.co/)).

**Reply by njames** [reply]
Thanks Keith, 

I had found those but I was wondering if there was a list somewhere of 'community' starter templates. 

I searched for jigsaw on packagist and I found a stack there. 

Not neatly organised as the built with site but useful.

Thanks for all you do.

**Comment by damiani** [comment]
Great, can't wait to see your new site!
